### PR TITLE
fix incorrect comparison error message

### PR DESCRIPTION
**Pull request by allencloud**
Signed-off-by: allencloud <allen.sun@daocloud.io>

What I did:
1. fix incorrect comparison error message

/cc @DirectXMan12

### CODE REVIEW

File: events/eventer.go [hunk 1]
```diff
@@ -123,12 +123,12 @@ func validateFlags() error {
 	var minFrequency = 5 * time.Second
 
 	if *argFrequency < minFrequency {
-		return fmt.Errorf("frequency needs to be greater than %s, supplied %s", minFrequency,
+		return fmt.Errorf("frequency needs to be greater than or equal to %s, supplied %s", minFrequency,
```
Comment (andyxning) [inline events/eventer.go]: How about `s/greater than or equal to/no less than`? @allencloud

File: events/eventer.go [hunk 2]
```diff
@@ -123,12 +123,12 @@ func validateFlags() error {
...
 			*argFrequency)
 	}
 
 	if *argFrequency > api.MaxEventsScrapeDelay {
-		return fmt.Errorf("frequency needs to be smaller than %s, supplied %s",
+		return fmt.Errorf("frequency needs to be smaller than or equal to %s, supplied %s",
```
Comment (andyxning) [inline events/eventer.go]: How about `s/greater than or equal to/no greater than`? @allencloud

**Comment by allencloud** [comment]
I have updated the PR. PTAL @andyxning 
Thanks

**Comment by andyxning** [comment]
Thanks for correct this! @allencloud 

/lgtm

**Pull request merged**